common/middleware.py

```diff
@@ -39,7 +41,9 @@ def __call__(self, request):
                     else:
                         flattened_content[key] = resource
                 elif resource is None:
-                    flattened_content[list(content['data'].keys())[index]] = resource
+                    flattened_content[
+                        list(content['data'].keys())[index]
+                    ] = resource
 
```

Comment:
@kola-er the line was too long, PEP8 lint issue ... that is why I broke it that way.